From 4543f4c5cbcf54f6302a41929411b09fdf96260f Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:03:18 -0400
Subject: [usb] wusbcore: add debug prints to reservation and channel change

Message-id: <1410887463-20674-127-git-send-email-dzickus@redhat.com>
Patchwork-id: 95349
O-Subject: [RHEL7 PATCH 126/591] usb: wusbcore: add debug prints to reservation and channel change
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit b4a9dfb02d932acd8a55ee96575e90f999a482de
Author: Thomas Pugliese <thomas.pugliese@gmail.com>
Date:   Fri Dec 20 11:45:02 2013 -0600

    usb: wusbcore: add debug prints to reservation and channel change

    This patch adds debug prints to the reservation and channel change
    sequence to help with debugging channel change problems.

    Signed-off-by: Thomas Pugliese <thomas.pugliese@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/wusbcore/pal.c b/drivers/usb/wusbcore/pal.c
index 59e100c..090f273 100644
--- a/drivers/usb/wusbcore/pal.c
+++ b/drivers/usb/wusbcore/pal.c
@@ -22,6 +22,7 @@ static void wusbhc_channel_changed(struct uwb_pal *pal, int channel)
 {
  struct wusbhc *wusbhc = container_of(pal, struct wusbhc, pal);
 
+ dev_dbg(wusbhc->dev, "%s: channel = %d\n", __func__, channel);
  if (channel < 0)
   wusbhc_stop(wusbhc);
  else
diff --git a/drivers/usb/wusbcore/reservation.c b/drivers/usb/wusbcore/reservation.c
index ead79f7..d5efd0f 100644
--- a/drivers/usb/wusbcore/reservation.c
+++ b/drivers/usb/wusbcore/reservation.c
@@ -51,6 +51,7 @@ static void wusbhc_rsv_complete_cb(struct uwb_rsv *rsv)
  struct uwb_mas_bm mas;
  char buf[72];
 
+ dev_dbg(dev, "%s: state = %d\n", __func__, rsv->state);
  switch (rsv->state) {
  case UWB_RSV_STATE_O_ESTABLISHED:
   uwb_rsv_get_usable_mas(rsv, &mas);
-- 
1.7.1